cypress/support/index.js

```diff
@@ -1,20 +1,11 @@
-// ***********************************************************
-// This example support/index.js is processed and
-// loaded automatically before your test files.
-//
-// This is a great place to put global configuration and
-// behavior that modifies Cypress.
-//
-// You can change the location of this file or turn off
-// automatically serving support files with the
-// 'supportFile' configuration option.
-//
-// You can read more here:
-// https://on.cypress.io/configuration
-// ***********************************************************
-
+/* global cy */
 // Import commands.js using ES2015 syntax:
 import './commands';
 
+cy.on('uncaught:exception', () => {
+  // returning false here prevents Cypress from
+  // failing the test
+  return false;
+});
```

Comment:
Is this just for debugging, while there are still ChunkLoadErrors?

Reply:
I want to see if the Firefox tests pass because of this.

Reply:
Sure, that makes sense.